client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -15,45 +15,81 @@ import LocationOnIcon from '@material-ui/icons/LocationOn';
 import useStyles from './useStyles';
 import ProfileImageList from './ProfileImageList';
 import { useAuth } from '../../context/useAuthContext';
-import { User } from '../../interface/User';
-import { mockLoggedInUser, mockProfileUser } from '../../mocks/mockUser';
+import { Sitter, User, Profile, Image } from '../../interface/User';
+import { getSitterProfile } from '../../helpers/APICalls/getSitterProfile';
+import { useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
 
 interface Props {
+  sitter: Sitter;
+  profile: Profile;
   user: User;
+  image: Image;
 }
 
+const initSitter: Sitter = {
+  profile: {
+    firstName: '',
+    lastName: '',
+    username: '',
+    location: '',
+    avgRating: 0,
+    user: {
```

Comment:
At later code u r using _sitter?.user?,_ , but this indicates sitter.profile.user ?